src/main/java/no/nav/familie/inntektsmelding/integrasjoner/dokgen/InntektsmeldingPdfDataMapper.java

```diff
@@ -0,0 +1,126 @@
+package no.nav.familie.inntektsmelding.integrasjoner.dokgen;
+
+import no.nav.familie.inntektsmelding.imdialog.modell.InntektsmeldingEntitet;
+import no.nav.familie.inntektsmelding.imdialog.modell.KontaktpersonEntitet;
+import no.nav.familie.inntektsmelding.imdialog.modell.NaturalytelseEntitet;
+import no.nav.familie.inntektsmelding.imdialog.modell.RefusjonPeriodeEntitet;
+import no.nav.familie.inntektsmelding.integrasjoner.person.PersonInfo;
+import no.nav.familie.inntektsmelding.koder.Naturalytelsetype;
+import no.nav.familie.inntektsmelding.koder.Ytelsetype;
+import no.nav.vedtak.konfig.Tid;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+import static no.nav.familie.inntektsmelding.integrasjoner.dokgen.InntektsmeldingPdfData.formaterDatoNorsk;
+
+public class InntektsmeldingPdfDataMapper {
+    public static InntektsmeldingPdfData mapInntektsmeldingData(InntektsmeldingEntitet inntektsmelding,
+                                                                String arbeidsgiverNavn,
+                                                                PersonInfo personInfo,
+                                                                String arbeidsgvierIdent) {
+        var imDokumentdataBuilder = new InntektsmeldingPdfData.Builder().medNavn(personInfo.mapNavn())
+            .medPersonnummer(personInfo.fødselsnummer().getIdent())
+            .medArbeidsgiverIdent(arbeidsgvierIdent)
+            .medArbeidsgiverNavn(arbeidsgiverNavn)
+            .medAvsenderSystem("NAV_NO")
+            .medYtelseNavn(inntektsmelding.getYtelsetype())
+            .medOpprettetTidspunkt(inntektsmelding.getOpprettetTidspunkt())
+            .medStartDato(inntektsmelding.getStartDato())
+            .medMånedInntekt(inntektsmelding.getMånedInntekt())
+            .medKontaktperson(mapKontaktperson(inntektsmelding.getKontaktperson()))
+            .medEndringIRefusjonsperioder(mapEndringIRefusjonsperioder(inntektsmelding.getRefusjonsPerioder()))
+            .medNaturalytelser(mapNauralYtelser(inntektsmelding.getNaturalYtelser()))
+            .medIngenBortfaltNaturalytelse(erIngenBortalteNaturalYtelser(inntektsmelding.getNaturalYtelser()))
+            .medIngenGjenopptattNaturalytelse(erIngenGjenopptatteNaturalYtelser(inntektsmelding.getNaturalYtelser()));
```

Comment:
Her må man vel til å filtrere på det boolske felter erBortalt , men er vel ikke kritisk for V1

Reply:
Det er sant, gjør det i neste commit.